src/myr/Myr.js

```diff
@@ -94,6 +98,9 @@ class Myr {
         this.cursor = {
             color: "red",
             texture: "",
+            materialType: MaterialType.SPECULAR,
+            roughness: 1.0,
```

Comment:
super nitpicky but this will forever bug me-- is there a particular reason you put 0.0/1.0 here instead of 0/1?

Reply:
Static typing habits. Used to floats being separate from integers. Yay javascript numbers